results.tex

```diff
@@ -171,61 +123,50 @@ \subsection{Transition Reprocessing Dynamics}
 	\begin{center}
 		\includegraphics[scale=0.7]{./images/french-transition/where_fuel.png}
 	\end{center}
-	\caption{This plot displays the timeseries of fuel loaded into \glspl{SFR}.
-			 The initial purple bars denote that the fuel is from reprocessing
-			 the previously used \gls{UOX} inventory. The peaks coincide
-			 with the new deployment of \glspl{SFR}.}
+	\caption{This plot shows the timeseries of fuel loaded into \glspl{SFR}.
+			 The plot has peaks during a period of aggressive deployment of \glspl{SFR}
+			 followed by an equilibrium at 150 \gls{MTHM}. The peaks reoccur with the
```

Comment:
reoccur is not a common word. recur is better. https://english.stackexchange.com/posts/33859/revisions